src/Compilers/CSharp/Test/Emit2/Semantics/RefReadonlyParameterTests.cs

```diff
@@ -8,6 +8,7 @@
 using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
 using Microsoft.CodeAnalysis.Test.Utilities;
 using Roslyn.Test.Utilities;
+using Roslyn.Utilities;
```

Comment:
The linked speclet change lists a few options for how to emit local functions. Please update it to show the selected design

Reply:
The options are under "Alternatives" section. The selected design is above: https://github.com/dotnet/csharplang/blob/e95d2d1c63a4b3d459d808e12589be90dd7ff772/proposals/ref-readonly-parameters.md#function-pointers